tests/riot.go

```diff
@@ -367,12 +369,18 @@ func (riot *Riot) Serve() {
 
 func newAttack(conf Config) {
 
-	cConf := client.DefaultConfig()
-	cConf.Endpoints = []string{conf.Endpoint}
-	cConf.APIKey = conf.APIKey
-	cConf.Insecure = conf.Insecure
+	// QED client
+	transport := http.DefaultTransport.(*http.Transport)
+	transport.TLSClientConfig = &tls.Config{InsecureSkipVerify: conf.Insecure}
+	httpClient := http.DefaultClient
+	httpClient.Transport = transport
+	client, err := client.NewHTTPClient(
+		client.SetHttpClient(httpClient),
+		client.SetURLs(conf.Endpoint[0], conf.Endpoint[1:]...),
+		client.SetAPIKey(conf.APIKey),
+		client.SetReadPreference(client.Any),
+	)
 
-	client, err := client.NewHTTPClientFromConfig(cConf)
```

Comment:
client.NewHTTPClientFromConfig it's near to be dead code, since it had 2 usages AFAIK.

Reply:
It's useful for testing.